<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayingDialog</class>
 <widget class="QDialog" name="PlayingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>540</y>
     <width>374</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="BTN_RW">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#BTN_RW
{
background image: url(:/images/images/rw.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BTN_Play">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/images/pause.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BTN_Stop">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/images/stop.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BTN_FF">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/images/ff.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BTN_UP">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/images/up.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BTN_Down">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/images/down.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
